tests/test_pb.py

```diff
@@ -25,7 +26,7 @@
 # The contents of a PB file with a header and one event.
 PB_CHUNK = RAW_PAYLOAD_INFO + b"\n" + RAW_EVENT
 # The actual contents of the above raw strings (PV name is not stored)
-EVENT = data.ArchiveEvent(PV, 264.65571148, 1422752399.004147078, 0)
```

Comment:
Has something changed here? Or is `EVENT`'s value never checked?

Reply:
The `ArchiveEvents` derived from an `ArchiveData` have values that look like this:
```array([264.6557114])```
because the array of `values` in the `ArchiveData` is 2D.
If you construct one like this, you get
```array(264.6557114)```
The `ArchiveEvent.__eq__``` used to evaluate to true for these two, but the bit I added to also check the enum string doesn't because it will fail if the values are different shapes. 
Probably I should be checking the enum options instead of the enum_string; then it would revert to the old behaviour.  I think I should do this anyway. But that leaves the question of whether the old behaviour was what we want or not.
p.s. the severity should always be a scalar I think, so I will change this in these lines.
Time to add some type hints!

Reply:
Tests are an important guide on how to use the code. If `value` and `values` are always arrays we should keep that consistent.